modules/nf-core/software/gatk4/markduplicates.nf

```diff
@@ -43,7 +43,7 @@ process GATK_MARKDUPLICATES {
     """
 }
 
-process GATK_MARKDUPLICATES_SPARK {
+process GATK4_MARKDUPLICATES_SPARK {
```

Comment:
the gatk version is not matching the other processes. Any reason to not update at least to 4.1.9? (or 4.2.0 if you already update the others?)